gammapy/utils/energy.py

```diff
@@ -189,6 +189,13 @@ def nbins(self):
         return self.size - 1
 
     @property
+    def lin_centers(self):
```

Comment:
Why introduce `lin_center`. For the application you use (and any use case I'm aware of), the existing `log_center` is better, no?